tools/cellpose/cellpose.xml

```diff
@@ -0,0 +1,154 @@
+<tool id="cellpose" name="Run Cellpose segmentation" version="@TOOL_VERSION@+galaxy@VERSION_SUFFIX@" profile="23.2">
+    <description>a generalist algorithm for cell and nucleus segmentation</description>
+    <macros>
+	<token name="@TOOL_VERSION@">3.0.1</token>
+	<token name="@VERSION_SUFFIX@">0</token>
+    </macros>
+    <requirements>
+	    <container type="docker">biocontainers/cellpose:3.0.1_cv1</container>
+	    <requirement type="package" version="0.22.0">scikit-image</requirement>
+    </requirements>
+    <stdio>
+        <exit_code range="1:" level="fatal" description="Error occurred. Please check Tool Standard Error"/>
+    </stdio>
+    <version_command>echo "@VERSION@"</version_command>
+    <command>
+         <![CDATA[
+        export CELLPOSE_LOCAL_MODELS_PATH='cellpose_models' &&
+        mkdir -p segmentation &&
+        ln -s '${img_in}' ./image.${img_in.ext} &&
+		
+        python '$__tool_directory__/cp_segmentation.py'
+            --inputs '$inputs'
+            --img_path ./image.${img_in.ext}
+            --img_format '${img_in.ext}'
+            --output_dir ./segmentation
+
+        ]]>
+
+    </command>
+    <configfiles>
+        <inputs name="inputs" />
+    </configfiles>
+    <inputs>
+        <param name="img_in" type="data" format="ome.tiff,tiff,jpeg,png" label="Choose the image file for segmention (usually after registration)"/>
+        <param name="model_type" type="select" label="Choose the pre-trained model type">
+            <option value="nuclei" selected="true">nuclei</option>
+            <option value="cyto">cyto</option>
+            <option value="cyto2">cyto2</option>
+            <option value="cyto3">cyto3</option>
+        </param>
+        <param argument="chan" value="0" type="integer" label="Type in the channel to segment" help="0=grayscale, 1=red, 2=green, 3=blue"/>
```

Comment:
why not a select here if there are only 3 options?

Reply:
yes, makes sense